doc/manual/config.md

```diff
@@ -1,5 +1,22 @@
 ### How to use the Config class
-The Config class is a class that handles whether or not certain features are enabled or disabled. The source is located at simplematrixbotlib/config.py
+The Config class is a class that handles whether certain features are enabled or disabled. The source is located at simplematrixbotlib/config.py
+
+#### Supported Values
```

Comment:
I like this, however I think that it should be located later on this page.

Reply:
I understand you added the setter method(s) to be used in the set config method. Since the values are publicly accessible, should we get rid of setters? Should there be getters? What is the design idea? I think the python way would be to be public. The "Supported Values" section could replace the "setting manually" (using setter methods) section since it's basically the same unless you want to do type validation or regex validation.

Reply:
We could deprecate the setters and replace it with public attributes as the recommended way of accessing the values.

Reply:
I think the best idea is using https://docs.python.org/3/library/functions.html#property decorators to achieve an attribute-like interface where it's needed but ability to implement custom validation (e.g. for regex) otherwise